# This message expresses the holonomic vehicle as combination of four motors(A,B,C,D)
Motor_desc  motor_A
Motor_desc  motor_B
Motor_desc  motor_C
Motor_desc  motor_D
